lighthouse-cli/cli-flags.js

```diff
@@ -33,8 +33,8 @@ function getFlags(manualArgv) {
           'lighthouse <url> --output=json --output-path=./report.json --save-assets',
           'Save trace, screenshots, and named JSON report.')
       .example(
-          'lighthouse <url> --disable-device-emulation --disable-network-throttling',
-          'Disable device emulation')
+          'lighthouse <url> --disable-device-emulation --throttling-method=provided',
```

Comment:
thoughts for another day: `--throttling-method=provided` -- this is kind of unfortunate wording if you really do just want throttling off. We could maybe make `none` or something that's basically like `provided`, though we probably don't want to make it a strict alias...

Reply:
yeah I also think this is awkward. Now with our laternification do we need our user to inform us if the throttling is off or provided?  If not, then I think we should rename `provided` (and maybe even `throttling-method`). 

Either way it's possible we have to introduce some CLI sugar that may be inconsistent with the config..

Reply:
Yeah I agree it's odd, there's no difference between `off` and `provided` I'm fine to rename to whatever in followup 👍